doc/userguide/rules/base64-keywords.rst

```diff
@@ -10,6 +10,8 @@ base64_decode
 
 Decodes base64 data from a buffer and makes it available for the base64_data function.
 
+It is recommended that the transform ``from_base64`` be used instead of ``base64_decode``.
```

Comment:
the `from_base64` here could be a `ref` to take the user directly to its section (for reference (heh): https://docs.ckan.org/en/2.9/contributing/documentation.html#cross-referencing-a-section-within-a-file)

Reply:
Will do.